test.sh

```diff
@@ -1,5 +1,11 @@
 export IS_CI=${1}
 
-docker-compose down --remove-orphans
+function cleanup {
+	docker-compose down --remove-orphans
+}
+
+trap cleanup EXIT
```

Comment:
If you are curious about `bash trap`, check out, https://github.com/vlab-research/fly/pull/77